### PR TITLE
Update documentation for curated packages

### PR DESCRIPTION
**Pull request by jhaanvi5**
*Description of changes:*
This PR updates README for curated packages (Prometheus, node-exporter, go-harbor, metallb).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/development/packages/update-helm-charts.md [hunk 1]
```diff
@@ -61,3 +71,24 @@ Note in some helm charts, fields above in `yaml` files are not hardcoded values
...
+Once the package/chart is in your personal registry test package
+
+Using helm
+
+```
+helm install {NAME} oci://{chart_registry}/{chart_repo} —version {chart_version} --set sourceRegistry={chart_registry}
+```
+Using EKSA CLI
+
+1. Generate local bundle using [generatebundlefile](https://github.com/aws/eks-anywhere-packages/tree/main/generatebundlefile) utility
```
Comment (vivek-koppuru) [inline docs/development/packages/update-helm-charts.md:91]: Do we have specific instructions to run here?

File: projects/metallb/metallb/README.md [hunk 2]
```diff
@@ -14,8 +14,23 @@ MetalLB is a virtual-ip provider for kubernetes services of type load balancer.
 1. Review [releases notes](https://metallb.universe.tf/release-notes/)
     * Any changes to the upstream configuration needs a thorough review + testing
     * Deprecation or removal of any protocol must be considered breaking 
-1. Update the `GIT_TAG` file to have the new desired version based on the upstream release tags.
-1. Verify the golang version has not changed. 
+1. Update the `GIT_TAG` and `GOLANG_VERSION` files to have the new desired version based on the upstream release tags.
+1. Review the patches under `patches/` folder and remove any that are either merged upstream or no longer needed.
```
Comment (vivek-koppuru) [inline projects/metallb/metallb/README.md:18]: What if we need to add any? Should we have a pointer on when that might be the case?

File: docs/development/packages/update-helm-charts.md (hunk 1 above)
Comment (jhaanvi5) [reply docs/development/packages/update-helm-charts.md:91]: Yes for those steps are provided in [README](https://github.com/aws/eks-anywhere-packages/tree/main/generatebundlefile#readme)

File: projects/metallb/metallb/README.md (hunk 2 above)
Comment (jhaanvi5) [reply projects/metallb/metallb/README.md:18]: Some of the cases where we need to add patch for package support are listed [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/docs/development/packages/update-helm-charts.md). I have linked this doc in this section in the README

`
To make changes to the patches folder, follow the steps mentioned [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/docs/development/packages/update-helm-charts.md#generate-patch-files)
`

**Review by vivek-koppuru — APPROVED** [review]
/approve

File: projects/prometheus/prometheus/README.md [hunk 3]
```diff
@@ -1,18 +1,35 @@
-![Version](https://img.shields.io/badge/version-v2.53.1-blue)
+![Version](https://img.shields.io/badge/version-v2.54.0-blue)
```
Comment (sp1999) [inline projects/prometheus/prometheus/README.md:1]: Why are we updating this version here in the README? It is still the old version v2.53.1 in the [GIT_TAG](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/prometheus/prometheus/GIT_TAG) file.

Comment (jhaanvi5) [reply projects/prometheus/prometheus/README.md:1]: This was supposed to get merged first
https://github.com/aws/eks-anywhere-build-tooling/pull/3671

**Pull request merged**